### ISSUE TITLE
training not smooth

### ISSUE THREAD

**Issue by jgmeyerucsd** [NREL/nfp#6]
Note really an issue with code - let me know if you prefer dm for this but I thought others might find it useful. 

I'm training a regression task that is very noisy in loss with my network tacked onto the end of the message passing part. The same training has very smooth loss with standard 1024-bit extended connectivity fingerprint and dense network with a couple hidden layers. 

Is there a way to freeze the early layers after they learn a chemical representation and then tune the bottom dense layers? any other ideas about how to make the loss more smooth?
![loss](https://user-images.githubusercontent.com/1245234/67050122-6bf32600-f0fd-11e9-82aa-551009659ab8.png)

**Comment by pstjohn** [comment]
> Can you please share that model and weights? 

Sure, but I'm not sure that would help with arbitrary chemical regression problems. We show in that paper that the transfer learning performance tends to correlate with how well the two problems are related. But those models are in this repo: https://github.com/pstjohn/opv_ml_models

>  is there a way to freeze the early layers

Yup, this would use the standard Keras syntax of freezing layers: https://keras.io/getting-started/faq/#how-can-i-freeze-keras-layers

> any other ideas about how to make the loss more smooth?

How large is your training / validation set? This could be a data set size issue, where your validation set is too small, and thus the loss is fairly noisy. You could also try using dropout or a smaller network to see if that improves things. Also make sure that the predictions for the training set are actually working, otherwise you might have miss-specified something along the way.
